### DISCUSSION TITLE
Use latest Prism changes with .Net Maui 7.0

### DISCUSSION THREAD

**Discussion by GUOLDEV** [PrismLibrary/Prism#2957] [.NET MAUI]
How can I use the latest Prism changes with .Net Maui 7.0? I tried it but it requires .net Maui 8.0.

Trying to use: https://github.com/PrismLibrary/Prism/commit/91beae021cfeefef4bc68bbcc89da29500d727b3
Works fine with this commit: https://github.com/PrismLibrary/Prism/commit/d8d47b8465038c18503761aaba6a68a9bdf35a0c

**Comment by dansiegel** [comment]
Prism will NOT be supporting net6.0 or net7.0. The changes that Prism.Maui has been waiting on for us to be in a position to release are only available in net8.0 and forward. You will need to update your project to target net8.0
